lib/overcommit/hook/pre_commit/bundle_check.rb

```diff
@@ -22,7 +22,8 @@ def run
 
       new_lockfile = File.read(LOCK_FILE) if File.exist?(LOCK_FILE)
       if previous_lockfile != new_lockfile
-        return :fail, "#{LOCK_FILE} is not up-to-date -- run `#{command.join(' ')}`"
+        return :fail, "#{LOCK_FILE} is not up-to-date -- run `#{command.join(' ')}` \
+                       or add the Gemfile and/or Gemfile.lock"
```

Comment:
This will result in a ton of whitespace between the command and this second line. Did you test this?

Reply:
Yeah, sorry, I didn't test..
I fixed this now! Sorry about that and tks